cmd/example/static.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Lekko Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"log"
+
+	client "github.com/lekkodev/go-sdk/client"
+)
+
+func main() {
+	// This will be generated by the lekko cli
+	var staticFeatures = map[string]map[string][]byte{
+		"default": {
+			"enable-puppy-dance": {
+				0x0a, 0x12, 0x65, 0x6e, 0x61, 0x62, 0x6c, 0x65, 0x2d, 0x70, 0x75, 0x70, 0x70, 0x79, 0x2d, 0x64,
```

Comment:
what if we use https://pkg.go.dev/embed and just copy proto files?
also, it will be nice to have something human-readable, maybe just copy json?
and git hash from config repo so it's easier to cross-reference

Reply:
I looked at embed a little bit, but I didn't come up with anything that felt better.  Checking out the config directory inside their repo would work, as would dropping the protofiles into a known subdirectory.  We'd still need to generate a file that would change a lot, but we'd also open more more possibilities of thrashing with their linters or opinions about where to put things.  I might just be missing how to make it feel clean end to end though.

The reason I didn't go for json is that our code already wants proto and since this will likely need to be run on startup, I didn't want to make things slow unless there was a really compelling reason to.

Reply:
The reason why I think that having protos in separate files is better, is because it will be easier to compare them with files from config repo.
About json, I was thinking to use proto, but also dump json on the side so humans can inspect it (could be useful for debugging). Similar idea to put git commit hash somewhere.

Reply:
I mean.. I agree that protobuf has bad dx but I don't think we need to fix that in this feature